code/switcher.py

```diff
@@ -55,6 +55,21 @@ def switcher_launch(path: str):
         """Launch a new application by path"""
         ui.launch(path=path)
 
+    def list_running():
```

Comment:
I'd like to try to maintain the convention of prefixing these with the file title.

e.g. switcher_show_running()